recipes/autograd/meta.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: autograd
+  version: "1.1.5"
+
+source:
+  fn: autograd-1.1.5.tar.gz
+  url: https://files.pythonhosted.org/packages/e5/70/e9c8db296d6ff33a17fec0ecaf11e3efbc85722d042365d01704046befef/autograd-1.1.5.tar.gz
```

Comment:
We prefer the `pypi.io` url, something like the following (assuming the use of the jinja tags):

```
source:
  fn: {{ name }}-{{ version }}.tar.gz
  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  md5: MD5_HERE
```